apps/monitoring/version-reporter/kustomization.yaml

```
@@ -7,4 +7,4 @@ resources:
   - renovate/renovate.yaml
   - docsoutdated/docs-outdated.yaml
   - platopsapps/platops-apps.yaml
-  - rbac.yaml
+  - ../../base/rbac.yaml
```

```suggestion
  - ../base/rbac.yaml
```
Almost, I think you have one too many ../ here